test/applications/cmd/icmp-responder-nse/main.go

```diff
@@ -81,9 +81,11 @@ func main() {
 				monitorServer := endpoint.MonitorServer(ctx)
 				logrus.Infof("Delaying 5 seconds before send update event.")
 				go func() {
-					<-time.After(5 * time.Second)
-					updateConnections(monitorServer)
-					logrus.Infof("Update event sended.")
+					for i := 0; i < 10; i++ {
```

Comment:
Why do we have to resend the connection update here?

Reply:
because at now we expect that the monitor-client will started in 5 sec after nse deployed (at removed line 84).
At now test failing because we have enabled opentracing on monitor-client side and it is not supported in tests yet (opentracing client can spent 5sec for check the server, you can take a look at nsm-monitor logs).
But I think we have some probability that it can happens and without opentracing.

Reply:
PR which fixes opentracing usage in tests https://github.com/networkservicemesh/networkservicemesh/pull/1644

Reply:
Good catch :)